opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorksErrorLine.tsx

```diff
@@ -32,6 +35,23 @@ const ConnectorWorksErrorLine: FunctionComponent<ConnectorWorksErrorLineProps> =
     return null;
   }
 
+  const entityListItem = (entity: ResolvedEntity) => {
+    const name = entity.representative?.main;
+    return (
+      <div>
+        {entity.entity_type ? (
+          <Tooltip title={name}>
+            <a href={`/dashboard/id/${entity.id}`} target="_blank" rel="noreferrer">
+              [{entity.entity_type}] {truncate(name, truncateLimit)}
+            </a>
+          </Tooltip>
+        ) : (
+          <div>{entity.standard_id}</div>
```

Comment:
what about
```suggestion
          <pre><ItemCopy content={entity.standard_id} variant={'wrap'} /></pre>
```
So that the standard id can be copied, that could help for investigation

Reply:
Try it, seems good, added